Amicabilibus+RemoveDuplicates/src/HarryPotterReader.java

```diff
@@ -0,0 +1,118 @@
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class HarryPotterReader {
+	
+	public static void main(String[] args) {
+		
+		Map<String, Integer> counted = countDuplicates(readFile("harry_potter.txt"));
```

Comment:
this is not needed for this assignment. You don't need to count how many of them are there. You just need the `readFile("harry_potter.txt")` and the `void removeDuplicate` method.